modules/tinymce/src/themes/silver/main/ts/ui/menus/menubar/SilverMenubar.ts

```diff
@@ -49,7 +49,8 @@ const factory: UiSketcher.SingleSketchFactory<SilverMenubarDetail, SilverMenubar
         text: m.text,
         fetch: (callback) => {
           callback(m.getItems());
-        }
+        },
+        context: '!readonly'
```

Comment:
Maybe something we need to discuss but we might need to have the menubar enabled in readonly but disable the menu items instead since if comments in readonly mode is enabled and the editor is in readonly and there is no add comment toolbar button and the user wants to add a comment using the menu item then there is no way to get to it?

Reply:
Wouldn't that be a behavioural change for readonly mode, as previously everything is disabled but now the buttons in the menubar are enabled?  However, for existing integrations, since the default menu items have `mode:design` only, no buttons should be enabled when it's in readonly mode.

Reply:
Yes it is a big change however we are not fully supporting comments in readonly mode if only the toolbar buttons works but menu items doesn't. If we are not fully supporting both UI elements then we need to make sure that people are aware that we only support comments though toolbar in readonly/comment only mode.

Reply:
Ah, sorry. The menubar buttons will be enabled when it's not in readonly, so the menu items in comments will have the same context `!readonly` and since we are going to register a new mode, so as long as it's not in `readonly` mode the menu items will be enabled. This applies to the toolbar buttons as well.

Reply:
Ok cool just wanted to double check. Right it checking the mode specifically not the readonly state. 👍